### PR TITLE
refactored variables containing master

### PR DESCRIPTION
**Pull request by gooney152**
#### What type of PR is this?
/kind cleanup
/wg-naming

#### What this PR does / why we need it:
updating the new name references master-to-control-plane

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
NONE

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by BenTheElder** [comment]
/ok-to-test

Unfortunately the project does not allow merge commits in PRs, so this PR cannot merge until that is fixed.
https://www.kubernetes.dev/docs/guide/github-workflow/ might be helpful

File: test/e2e/framework/resource_usage_gatherer.go [hunk 1]
```diff
@@ -603,21 +603,21 @@ type kubemarkResourceUsage struct {
...
-func getKubemarkMasterComponentsResourceUsage() map[string]*kubemarkResourceUsage {
+// getKubemarkControlPlaneComponentsResourceUsage returns the resource usage of kubemark which contains multiple combinations of cpu and memory usage for each pod name.
+func getKubemarkControlPlaneComponentsResourceUsage() map[string]*kubemarkResourceUsage {
 	result := make(map[string]*kubemarkResourceUsage)
 	// Get kubernetes component resource usage
-	sshResult, err := getMasterUsageByPrefix("kube")
+	sshResult, err := getControlPlaneUsageByPrefix("kube")
 	if err != nil {
-		Logf("Error when trying to SSH to master machine. Skipping probe. %v", err)
+		Logf("Error when trying to SSH to controlplane machine. Skipping probe. %v", err)
```
Comment (BenTheElder) [inline test/e2e/framework/resource_usage_gatherer.go:620]: tiny non-blocking nit: when it's not a variable name we should leave a space between the words
```suggestion
		Logf("Error when trying to SSH to control plane machine. Skipping probe. %v", err)
```

**Review by BenTheElder — COMMENTED** [review]
/approve

**Comment by gooney152** [comment]
@BenTheElder

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/kubernetes/pull/103569/commits/16b924e9d6fe99ad4d9669201d41f12fe7e2bd48 and https://github.com/kubernetes/kubernetes/pull/103569/commits/3f5c70a930b6577e11c2c5c21bfe00837a346955 prevent this from being mergeable (see: https://github.com/kubernetes/kubernetes/pull/103569#issuecomment-876646290)

Otherwise these changes look good, I've already approved but I may be out the rest of this and next week so probably someone else will need to `/lgtm` (any member of the project) once the robot is no longer blocking merge due to the merge commits.

Thanks for the PR!

**Pull request closed without merging**